magicctapipe/scripts/lst1_magic/semi_automatic_scripts/setting_up_config_and_dir.py

```diff
@@ -0,0 +1,740 @@
+"""
+This script facilitates the usage of
+"magic_calib_to_dl1.py". This script is more like a
+"manager" that organizes the analysis process by:
+1) Creating the necessary directories and subdirectories.
+2) Generating all the bash script files that convert the
+MAGIC files from DL0 to DL1.
```

Comment:
MAGIC _Y_ files are not exactly DL0, because they are already calibrated, maybe write "from _Y_ MAGIC format to DL"

Reply:
fixed